pmlc/dialect/stripe/types.h

```diff
@@ -135,7 +135,7 @@ class TensorType : public Type::TypeBase<TensorType, Type, TensorTypeStorage> {
   Type getElementType() const { return getImpl()->elementType; }
 
   /// Return the rank.
-  int64_t getRank() const { return getImpl()->shape.size(); }
+  uint64_t getRank() const { return getImpl()->shape.size(); }
```

Comment:
These should remain `int64_t` so that it's easier to convert to/from `RankedTensorType` (which is what the tile dialect uses).

Reply:
Done